### ISSUE TITLE
Suggestions for unmistakable naming

### ISSUE THREAD

**Issue by Pilkja** [RealRaven2000/FiltaQuilla#48]
Just a few suggestions **for the German interface:**
In the prefs / Filter actions: 
![filtaquilla DE](https://user-images.githubusercontent.com/59780183/72211673-a2a7c500-34cf-11ea-9bc2-4ae88fc3dba9.png) 
![03](https://user-images.githubusercontent.com/59780183/72211674-adfaf080-34cf-11ea-9bde-612e4076ec5e.png)

**And for English:**
(this image is from your website, though - may be different from the current English interface) 
![filtaquilla EN](https://user-images.githubusercontent.com/59780183/72211694-f87c6d00-34cf-11ea-856b-398fa290018c.png)

**Comment by RealRaven2000** [comment]
Ok, I had a look. fixed most of them, but I am still not sure whether "Schlagwort der Ersten Mail des Themas" is correct. I want to say "Tag of the first mail of thread = (Thread Head)" which is hard to do in a succinct manner in German, I believe "Schlagwort der Ersten Mail der Unterhaltung" is more correct, I feel "Thema" would be equivalent to the "Subject" which is confusing. I don't think there is a good German word for "thread", so I opted for "Unterhaltung". I could be persuaded to replace "aller Mails", hoping the user understands that this means all mails of the thread will be affected.

![image](https://user-images.githubusercontent.com/6314674/72219488-8f800e00-353e-11ea-9f04-dc94228a2659.png)

![image](https://user-images.githubusercontent.com/6314674/72219495-ab83af80-353e-11ea-895f-4c423968b5da.png)

Thunderdbird does not thread emails according to subject (like Outlook used to do for a long time) but according to some other header/s.

Also, "Run Program" is a better (more distinct) way of expressing the difference to "Launch File". I think Launch File will use the-OS specific integration to start the associated program, whereas "Run Program" is actually starting a binary (in windows usually exe[cutable] or compiled.com), which is definitely different and may be a lot more harmful. In German, I have translated these to "Datei ausführen" and "Programm ausführen"
